deploy.yml

```diff
@@ -158,25 +159,26 @@ jobs:
 
           eq_author_tag=$(cat ../tag)
           echo "{\"APPLICATION_VERSION\":\""$eq_author_tag"\"}" > ../../build_args/args
+          cp -R ../eq-author-app/* ../../extracted-eq-author-app
```

Comment:
What's the reason for having to copy everything into the direction above? Isn't it simpler to go up one directory for the `build_args_file`?

Reply:
We are 2 folders deep and the output folders get created at the root

Reply:
We have to copy it as we extract it from the `tar.gz`

Reply:
maybe we dont need to `cd eq-author-app` thou